src/extension.ts

```diff
@@ -1,40 +1,21 @@
-// The module 'vscode' contains the VS Code extensibility API
-// Import the module and reference it with the alias vscode in your code below
-import * as vscode from 'vscode';
-import { graphql } from '@octokit/graphql';
+import { AuthenticationSession, ExtensionContext } from 'vscode';
+import { authentication, window} from 'vscode';
```

Comment:
you've got two imports from vscode. I recommend either:

1. using `import * as vscode from 'vscode';` and then use `vscode.*` everywhere
2. combining these